test/models/identity_header_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class IdentityHeaderTest < ActiveSupport::TestCase
+  setup do
+    @b64_identity = {
+      'entitlements': {
+        'smart_management': {
+          'is_entitled': true
+        }
+      },
+      'identity': {
+        'account_number': '293912'
+      }
+    }
+  end
+
+  test 'is valid if smart_management entitlement provided' do
+    assert IdentityHeader.new(Base64.encode64(@b64_identity.to_json))
```

Comment:
Technically, I think we should be using [`strict_encode64`](https://ruby-doc.org/stdlib-2.5.3/libdoc/base64/rdoc/Base64.html#method-i-strict_encode64) without line feeds. Right?